fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -151,11 +151,11 @@ def self.parse_response(response)
       private_class_method :parse_response
 
       def self.description
-        'Upload a new build to [TestFairy](https://www.testfairy.com/)'
+        'Upload a new build to [TestFairy](https://saucelabs.com/products/mobile-testing/app-betas)'
```

Comment:
Actually the tests on CI complain with:
```
[17:34:04]: ▸ Provided description for 'testfairy'-action is too long
```

Let's try moving part of this to `self.details`…

```suggestion
        'Upload a new build to SauceLabs' TestFairy'
```